README.md

```diff
@@ -0,0 +1,41 @@
+Public pre-commit hooks developed at Ponte Energy Partners
+
+
+## Available hooks
+
+### `verify-git-email`
+
+This hook verifies that a committer's email address matches a domain.
+
+This hook can serve two use cases:
+- Ensure people hide their private e-mail addresses, e.g. by using the respective [GitHub setting](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address). You can set `--domain=users.noreply.github.com` in that case. Note that GitHub provides a way to [block pushes that contain private email addresses](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/blocking-command-line-pushes-that-expose-your-personal-email-address).
+
+- Ensure that people within an organisation commit with their org email address instead of their private address.
+
+### `mypy-with-poetry`
+
+Runs mypy with your system's `poetry` executable (through `language: system`). The advantage over the [mypy mirror repo](https://github.com/pre-commit/mirrors-mypy#using-mypy-with-pre-commit) is that the virtual environment configured with poetry gets used instead of one that pre-commit manages (with `language: python`). That way, you don't need to configure `additional_dependencies:` for the mypy hook to work. The hook `mypy-with-poetry` requires `files` to be specified in `pyproject.toml`, e.g. like this:
+
+    ```toml
+    [tool.mypy]
+
+    files = "."
```

Comment:
This could also be hard-coded in `.pre-commit-hooks.yaml`: `entry: poetry run mypy .`

I prefer the config file approach, since users may adjust that to suit their project's needs.

Here's a link to [MyPy docs](https://mypy.readthedocs.io/en/stable/config_file.html#confval-files)